Formula/smimesign.rb

```diff
@@ -0,0 +1,21 @@
+class Smimesign < Formula
+  desc "S/MIME signing utility for use with Git"
+  homepage "https://github.com/github/smimesign"
+  url "https://github.com/github/smimesign/archive/0.0.1.tar.gz"
+  sha256 "13789368b1379aa2ad52bce5b855b56ca88f684ed0dff42adf107ee9738187b0"
+  depends_on "go" => :build
```

Comment:
Can this specify the Go version? This requires Go>=1.10.4.

Reply:
No, it will build with whatever version `go` is, currently `1.11`.